src/store/printer/getters.ts

```diff
@@ -151,6 +153,7 @@ export const getters: GetterTree<PrinterState, RootState> = {
           ? current / progress
           : current
         remaining = total - current
+        endTime = endTime + remaining
```

Comment:
It seems like this (`case 'file'`) is the only situation in which `endTime` would be meaningful.  If that's not so then I'll update the other cases as needed!

Reply:
Again, on digging I see that this isn't the full story. Will update tomorrow!